draft-meynell-panrg-scion-research-questions.md

```diff
@@ -471,11 +473,18 @@ Is reverse path refresh a relevant problem?
 -->
 
 # Interfaces for Path Awareness
+The questions posed here are:
 
-* IPv6 in the Data Plane
-* SCION-IP translation
+* Are there any existing protocols/solutions that can carry path aware traffic?
```

Comment:
Maybe we should be a bit more precise, I think there are a lot of solution for path aware traffic. How about:
`- Are there any existing protocols/solution that can be adapted to carry SCION traffic?`
?